bower.json

```diff
@@ -28,23 +28,45 @@
     "wct.conf.js"
   ],
   "dependencies": {
-    "iron-icon": "PolymerElements/iron-icon#^1.0.0",
-    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
-    "paper-progress": "PolymerElements/paper-progress#^1.0.0",
-    "paper-ripple": "PolymerElements/paper-ripple#~1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.7.0"
+    "iron-icon": "^1.0.0",
+    "iron-iconset-svg": "^1.0.0",
+    "paper-button": "^1.0.0",
+    "paper-icon-button": "^1.0.0",
+    "paper-progress": "^1.0.0",
+    "paper-ripple": "PolymerElements/paper-ripple#^1.0.0",
+    "paper-styles": "^1.0.0",
+    "polymer": "^1.7.0"
   },
   "devDependencies": {
-    "elements-demo-resources": "Vaadin/elements-demo-resources#master",
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.0.0",
+    "iron-component-page": "^1.0.0",
+    "iron-demo-helpers": "^1.0.0",
+    "iron-test-helpers": "^1.0.0",
+    "elements-demo-resources": "vaadin/elements-demo-resources#master",
     "mock-http-request": "philikon/MockHttpRequest#master",
-    "paper-toast": "PolymerElements/paper-toast#^1.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "Polymer/web-component-tester#^5.0.0"
+    "paper-toast": "^1.0.0",
+    "web-component-tester": "^6.0.0-prerelease.7",
+    "webcomponentsjs": "^0.7.0"
+  },
+  "variants": {
+    "polymer2": {
```

Comment:
It looks like your variants are backward. Should be `1.x` variant and mainline should be 2.0. After an official version of Polymer 2.0 is tagged, using `^1.9||^2.0` (this was the syntax discussed as the way to move forward post 2.0 tag of polymer)

https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements#dependency-variants